### DISCUSSION TITLE
keybord broken

### DISCUSSION THREAD

**Discussion by hpvd** [terminusdb/terminusdb#2046] [General]
you have published a really interesting blog post about the cli, but there seem to be some strange problems with the keyboard when typing it :-)
see: https://terminusdb.com/blog/terminusdb-from-the-command-line-introducing-tdb-cli/
![2023-11-28_15h57_52](https://github.com/terminusdb/terminusdb/assets/5681880/339f712e-2238-4c25-b93f-4513af2dcc37)

**Comment by GavinMendelGleason** [comment]
Yikes!

**Reply by GavinMendelGleason** [reply]
Hopefully fixed now.

**Reply by hpvd** [reply]
yeah looks great.
just a very last t seems to be disappeared... `you wan to start another TerminusDB`
